### PR TITLE
Add generated test coverage badge to gitignore

### PR DESCRIPTION
**Pull request by shyndman**
This file is generated as a result of a `pub run test_coverage`, and is unneeded.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,6 +31,7 @@ Generated.xcconfig
 
 coverage/
 .test_coverage.dart
+coverage_badge.svg
```
Comment (pavanpodila) [inline .gitignore]: How is this relevant to us ?

Comment (shyndman) [reply .gitignore]: If you run test_coverage locally, it gets generated. 🤷‍♂️

Comment (pavanpodila) [reply .gitignore]: Umm....ok
